linux/registerMediaKeys.js

```diff
@@ -1,4 +1,13 @@
 const DBus = require('dbus');
+const { ipcMain } = require('electron');
+
+let status = 'Stopped';
```

Comment:
I have a small issue with this approach. The embedded player already has a state (https://developers.google.com/youtube/iframe_api_reference search for `getPlayerState`) which I treat as the source of truth. Adding another state might cause syncing problems and the need to always maintain it for every new piece of code. In the web-app, I'm also trying to avoid having a duplicated state and I'm only relying on the info I get from the player. 

Perhaps a better approach will be to check if there is a track loaded (as this is the root of the issue)
You can achieve it with
```
let track = null

ipcMain.on('win2Player', (e, args) => {
  if(args[0] === 'trackInfo') {
    track = args[1] // in the future we might need to do something with the track.
  }
})
```